src/libLookup/Lookup.cpp

```diff
@@ -3127,6 +3131,11 @@ void Lookup::SendTxnPacketToNodes(uint32_t numShards) {
 
       LOG_GENERAL(INFO, "Transaction number generated: " << transactionNumber);
 
+      if (m_txnShardMap[i].size() == 0 && mp[i].size() == 0) {
```

Comment:
Can use m_txnShardMap[i].empty() instead of m_txnShardMap[i].size() == 0, it is more efficient